sssom/cli.py

```diff
@@ -114,38 +113,41 @@ def crosstab(input, output, transpose, fields):
 @main.command()
 @click.option('-o', '--output')
 @click.option('-t', '--transpose/--no-transpose', default=False)
+@click.option('-v', '--verbose/--no-verbose', default=False)
```

Comment:
there is a way in click to make verbose a global option.. we could do this.. or it may just complicate